easybuild/easyconfigs/i/intel-compilers/intel-compilers-2021.3.0.eb

```diff
@@ -1,6 +1,3 @@
-##
-# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
-##
```

Comment:
@SebastianAchilles Let's restore this bit? I know it matters for @robert-mijakovic